pkg/scraper/jobcenter_scraper.go

```diff
@@ -2,37 +2,66 @@ package scraper
 
 import (
 	"fmt"
+	"regexp"
 	"strconv"
 
+	"github.com/b-open/jobbuzz/internal/util"
 	"github.com/b-open/jobbuzz/pkg/model"
 	"github.com/gocolly/colly"
 )
 
+const (
+	delta = 30
```

Comment:
`delta` might be a bit vague to what it means. Maybe use `pageSize` or something like that.